files/default/neutron-ha-tool.py

```diff
@@ -963,6 +963,27 @@ def pick(self):
         return self.agents_by_id[agent_id]
 
 
+class SingleAgentPicker(object):
+    agent_keys = ('id', 'host')
+    agent_selection_value = None
+
+    def __init__(self, qclient, agents):
+        self.agents = agents
+
+    def find_agent(self, value):
```

Comment:
Right, I guess the best would be to enable users to specify explicitly what the string refers to.